ts/features/bonusVacanze/screens/BonusInformationScreen.tsx

```diff
@@ -236,6 +254,17 @@ const BonusInformationScreen: React.FunctionComponent<Props> = props => {
   );
 };
 
+const mapStateToProps = (state: GlobalState) => {
+  const profile = pot.toUndefined(profileSelector(state));
+  const hasAnotherBonus = fromNullable(profile).fold(false, p => {
```

Comment:
Having one place where this information is provided have some benefits:

- [DRY](https://it.wikipedia.org/wiki/Don%27t_repeat_yourself)
- [SSOT](https://en.wikipedia.org/wiki/Single_source_of_truth)
- more testable